internal/controller/clusterinstance_controller.go

```diff
@@ -39,10 +42,18 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 
 	"github.com/stolostron/siteconfig/api/v1alpha1"
+	ai_templates "github.com/stolostron/siteconfig/internal/templates/assisted-installer"
+	ibi_templates "github.com/stolostron/siteconfig/internal/templates/image-based-installer"
 )
 
 const clusterInstanceFinalizer = "clusterinstance." + v1alpha1.Group + "/finalizer"
 
+// Disaster recovery constants
+const (
+	acmBackupLabel      = "cluster.open-cluster-management.io/backup"
+	acmBackupLabelValue = "true"
```

Comment:
The value can be anything, even an empty value; the CR is backed up as long as the acmBackupLabel exists
Use the **cluster-activation** value if the CR needs to be restored only at cluster activation time.

Reply:
Thanks for the additional information @birsanv! The SiteConfig Operator would fall under the "value can be anything" category. So I will leave value to be `"true"` unless an empty string is preferrred?

Reply:
@sakhoury there is no default value so you can use "true" if this is what you prefer
I found it a bit confusing when the boolean value "true" is used as it could imply that a "false" value would not result in the CR being backed up. But again, in the end is your choice, I just wanted to make sure you are aware of this

Reply:
That's a great point! I think I will change this to an empty string to avoid potential confusion that may arise around the boolean values...